pkg/api/login_test.go

```diff
@@ -609,7 +609,7 @@ func (r *loginHookTest) LoginHook(loginInfo *models.LoginInfo, req *models.ReqCo
 	r.info = loginInfo
 }
 
-func TestLoginPostRunLokingHook(t *testing.T) {
+func TestLoginPostRunValingHook(t *testing.T) {
```

Comment:
Intentional change? Probably a Typo in the first place.

Reply:
Hah, good catch. Reverted it.